python/examples/examples/vast.py

```diff
@@ -0,0 +1,89 @@
+from typing import List
+
+import pandas as pd
+
+from datashaper import ExecutionGraph, TableContainer, VerbInput
+
+dtype = {"date(yyyyMMddHHmmss)": "string"}
+vast1 = pd.read_csv("../data/vast/csv-1700-1830.csv", dtype=dtype)
```

Comment:
We should either bundle some sample data (with no redistribution issues) in the repo, or provide instructions for getting this data and locating it relative to the notebook

Reply:
I totally agree, I debated even putting it in this PR, but I think it demonstrates the API changes well and gives you, me and Alonso a strawman to work with